oemof/outputlib/processing.py

```diff
@@ -86,6 +86,7 @@ def create_dataframe(om):
     # on which dimension the variable/parameter has (scalar/sequence).
     # columns for the oemof tuple and timestep are created
     df['oemof_tuple'] = df['pyomo_tuple'].map(get_tuple)
+    df = df[df['oemof_tuple'].map(lambda x: x!=None)]
```

Comment:
One thing I am not completely sure about is this little hack that I used to avoid an error in the outputlib. The constraints produced by pyomo's Piecewise are labelled differently in the LP-file than oemof normally expects. This line excludes those, but I am not sure if this is the best way and if it does not interfere somewhere.